tests/integration_tests/test_upgrade.py

```diff
@@ -141,7 +140,7 @@ def test_cosmovisor_upgrade(custom_ethermint: Ethermint):
     cli = custom_ethermint.cosmos_cli()
 
     # block should pass the target height
-    wait_for_block(cli, target_height + 1, timeout=480)
+    wait_for_block(cli, target_height + 2, timeout=480)
```

Comment:
why `+2`?

Reply:
just to align with [here](https://github.com/evmos/ethermint/blob/7f196ce451131ab5346d1635a48e435c69f592ba/tests/integration_tests/test_upgrade.py#L172)

Reply:
ditto, comment